com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -159,6 +194,31 @@ private static DynamicResolutionHandler GetOrCreateDrsInstanceHandler(Camera cam
             return instance;
         }
 
+        /// <summary>
+        /// The scheduling mechanism to apply upscaling.
+        /// </summary>
+        public enum UpsamplerScheduleType
+        {
+            /// <summary>
+            /// Indicates that upscaling must happen before post processing.
```

Comment:
You should include in the description what is the difference :), i.e with current description it is not possible to  know why chose one option rather than the others